### DISCUSSION TITLE
Setting runner group when installing runner --unattended

### DISCUSSION THREAD

**Discussion by mv185095** [community/community#25909] [Actions]
<p>With recent additions of groups for runners we would like to also make use of them for our automatically created runners. When setting the runner manually the <code>./config.sh</code> asks for the group yet I do not see the<br>
option in the help for setting it on the command line along with <code>--unattended</code>. Is it just undocumented or currently not possible to set programatically?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/mv185095">@mv185095</a>,</p>
<p>Currently it’s not supported to set the runner group programtically, you can set it in the UI:<br>
<a href="https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups</a></p>
<p>It’s supported to set the runner as a <a href="https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service" rel="nofollow noopener">service</a>.</p>
<p>Thanks</p>
